.github/workflows/azure-static-web-apps-purple-plant-006657c1e.yml

```diff
@@ -0,0 +1,46 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - Assignment1

```

Comment:
To answer your question in teams: Change this to `Assignment2` to deploy your next assignment!

Reply:
Great, thank you Meghan!